documentation/templates/theme/initiatives/epics/stories/tasks/task_template.md

```diff
@@ -1,15 +1,17 @@
-Task Title: [Short, descriptive title for the tasks]
+Tasks are a key component of agile project management, breaking down user stories into smaller, actionable steps that can be assigned and tracked. 
 
-User Story: [Reference the user story that this task supports]
+Task Title: Set up Google Analytics for website
 
-Description: [Describe the task in more detail. What needs to be done? What are the acceptance criteria? What are the dependencies?]
+User Story: As a website owner, I want to track website traffic so that I can improve my website's performance.
```

Comment:
Another user story to think about for "Set up Google Analytics for website" could be that the website owner would like to create it to track user behavior, monitor its performance and make informed and strategic decisions based off the data.